boltdesignsystem/bolt
# DS-902: Icon in the menu comp doesn't display in proper place

## Jira

https://pegadigitalit.atlassian.net/browse/DS-902

## Summary

When the `icon_after` prop is used, the icon displays after the menu's text.

## Details

- The `_menu-item.twig` template was updated to make the `icon_after` prop work as expected
- The `_menu-item.scss` was updated to remove unnecessary grid gap at the end of the menu item
- The doc page **menu use case icons** was updated with the `icon_after` examples

## How to test

Pull the branch. Go to `pattern-lab/?p=components-menu-use-case-icons` and confirm that icons display properly before and after the text.

## Release notes

The bug in the menu component was fixed to make the icon display properly after the menu's text.
